contracts/RelayHub.sol

```diff
@@ -321,25 +335,29 @@ contract RelayHub is IRelayHub {
         emit TransactionRelayed(msg.sender, from, recipient, functionSelector, status, charge);
     }
 
-    function getChargedAmount(uint256 gas, uint256 gasPrice, uint256 fee) private pure returns (uint256) {
-        // The fee is expressed as a percentage. E.g. a value of 40 stands for a
-        // 40% fee, so the recipient will be charged for 1.4 times the spent
-        // amount.
-        return (gas * gasPrice * (100 + fee)) / 100;
+    struct AtomicData {
```

Comment:
Unfortunately no. Solidity keeps complaining about stack size in this method if we don't move some values to memory (what we're doing by moving them to a `memory` struct). The upside is that, since it is used internally in a function, it does not require adding the EncoderV2.